### ISSUE TITLE
Check personal vocab for accessibility & edibility

### ISSUE THREAD

**Issue by bilbe** [Hedera-Lang-Learn/hedera#347]
On the Personal Vocab page, a user should be able to edit an entry in place and should follow page should follow accessibility guidelines.
